tests/unit/sagemaker/image_uris/test_data_wrangler.py

```diff
@@ -42,14 +42,12 @@
 
 
 def test_data_wrangler_ecr_uri():
-    for region in regions.regions():
-        if region in DATA_WRANGLER_ACCOUNTS.keys():
-            actual_uri = image_uris.retrieve("data-wrangler", region=region)
-
-            expected_uri = expected_uris.algo_uri(
-                "sagemaker-data-wrangler-container",
-                DATA_WRANGLER_ACCOUNTS[region],
-                region,
-                version="1.x",
-            )
-            assert expected_uri == actual_uri
+    for region in DATA_WRANGLER_ACCOUNTS.keys():
```

Comment:
Same here. I think removing regions.region here is not testing what we intended in theses tests.

All of these test creates image_uris from boto regions and tests on SDK that if we can create that URI and support that or not. 

Otherwise we just creating two image uris from same source `DATA_WRANGLER_ACCOUNTS`